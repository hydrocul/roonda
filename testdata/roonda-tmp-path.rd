(sh)

(cd (ref ROONDA_TMP_PATH))
(touch "abc.txt")
(ls -a)
(touch "def.txt")
(ls -a)

