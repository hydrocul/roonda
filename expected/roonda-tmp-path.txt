.
..
abc.txt
c5b5ddbfb363357b8b82006237993854f082f736.sh
.
..
abc.txt
c5b5ddbfb363357b8b82006237993854f082f736.sh
def.txt
